storyblok/app-extension-auth
# refactor: Cookie Utility Functions

## What

Issue: EXT-1513

Refactoring of cookie utility functions. No behavioural changes.

- Makes `src/utils/` a module by adding `src/utils/index.ts`. This will allow us to shorten the import paths in other files that use utility functions.
- Moves the utility functions that relates to coookies into the `src/utils/` module. These are standalone functions that work on their own.
- Enforces consitent naming conventions. Files are called the same as the token they export. 
- `changedCookieHeaderValue` and `expiredCookieHeaderValue` are renamed and moved out into their own files.


## Why

This cleanup will make it easier to later replace grant with our own OAuth handlers, which will be necessary to make apps work on US spaces.

## How to test


The library should build and tests should pass:

```
yarn build
yarn test
```

You can also link this package to the Next.js starter project and see that it runs smoothly.
